RaccoonPG.py

```
@@ -0,0 +1,389 @@
+        self.inCombat = False
+        self.money = 0
+        self.LVL = 1
+
+
+        self.set_slow_xp(EXPcap, LVL)
+
+    def deal_damage(self, enemy):
+        damage = Animal.damage_formula(self, enemy, self.POWER, self.LVL, self.ATTK, self.CRIT)
+        enemy.HP -= damage
+        if enemy.HP <= 0:
+            enemy.inCombat == False
```

Ruff (0.14.0)

Avoid equality comparisons to `False`; use `not enemy.inCombat:` for false checks

Replace with `not enemy.inCombat`

(E712)